src/com/facebook/buck/maven/ArtifactConfig.java

```diff
@@ -0,0 +1,102 @@
+package com.facebook.buck.maven;
+
+import com.google.common.base.Charsets;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.annotations.SerializedName;
```

Comment:
I /think/ the Buck project is trying to standardize on Jackson, but @Coneko would know for sure.